examples/CustomCollectionView/Sample/AppDelegate.m

```diff
@@ -23,7 +23,7 @@ @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
-  self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
+  self.window = [[UIWindow alloc] initWithFrame:UIScreen.mainScreen.bounds];
```

Comment:
This file had not been modified post-Texture, so this file was checked against the new license text (hack did not apply.)